Explorer/Assets/Scripts/SceneRuntime/Factory/SceneRuntimeFactory.cs

```diff
@@ -21,16 +24,29 @@ public enum InstantiationBehavior
             SwitchToThreadPool,
         }
 
-        private readonly JsCodeResolver codeContentResolver;
-        private readonly Dictionary<string, string> sourceCodeCache;
+        private readonly IWebJsSources webJsSources;
+        private readonly IJsSourcesCache jsSourcesCache;
 
         private static readonly IReadOnlyCollection<string> JS_MODULE_NAMES = new JsModulesNameList().ToList();
         private readonly IJsSceneSourceCode jsSceneSourceCode = new IJsSceneSourceCode.Default();
 
         public SceneRuntimeFactory(IWebRequestController webRequestController)
         {
-            codeContentResolver = new JsCodeResolver(webRequestController);
-            sourceCodeCache = new Dictionary<string, string>();
+            jsSourcesCache = new IJsSourcesCache.Fake();
```

Comment:
Maybe place it in the `#else` branch? 
Overall it's a bit strange that we use `Fake` class (that we normally use for debug/test) in the production flow.